tests/regression/lp-1595444/task.yaml

```diff
@@ -4,15 +4,9 @@ details: |
     This task checks the behavior of snap-confine when it is started from
     a directory that doesn't exist in the execution environment (chroot).
 
-systems:
-    # This test only applies to classic systems
-    - -ubuntu-core-*
-    # No confinement (AppArmor, Seccomp) available on these systems
-    - -debian-*
-    - -fedora-*
-    - -opensuse-*
-    - -arch-*
-    - -amazon-*
+# This test only applies to classic systems
```

Comment:
Same comment as above, if we collapse it I think it would be good to add a prefix to the comment so that the reads understands for what system the comment is.